lumigator/python/mzai/backend/backend/tests/api/routes/test_datasets.py

```diff
@@ -113,3 +113,35 @@ def test_experiment_format_validation(
             files={"dataset": ("dataset.csv", d)},
         )
         assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+def test_upload_csv_delete(app_client: TestClient):
+    response = app_client.get("/health")
+    assert response.status_code == status.HTTP_200_OK
```

Comment:
I don't think we need to test the health endpoint here

Reply:
You'd be surprised (or not!) but this one is helpful as a sanity check: it helped see a 404 I was getting was due to me messing with how I was handling URLs, so it might help someone else.